package/kernel/linux/modules/leds.mk

```diff
@@ -131,6 +131,22 @@ endef
 $(eval $(call KernelPackage,leds-apu))
 
 
+define KernelPackage/leds-lp55xx-common
+  SUBMENU:=$(LEDS_MENU)
+  TITLE:=LED driver common code for LP55XX controllers
+  #DEPENDS:=+kmod-i2c-core +kmod-regmap-i2c
+  KCONFIG:=CONFIG_LEDS_LP55XX_COMMON
+  FILES:=$(LINUX_DIR)/drivers/leds/leds-lp55xx-common.ko
+  AUTOLOAD:=$(call AutoLoad,50,leds-lp55xx-common,1)
```

Comment:
This looks wrong, why is it autoloaded at all and why is it required in preinit?
It should get autoloaded by modules that consume symbols from it anyway.